src/xcode/ENA/ENA/Source/Workers/TracingStatusHistory.swift

```diff
@@ -91,7 +91,10 @@ extension Array where Element == TracingStatusEntry {
 	///
 	/// - parameter since: Date to use as the baseline. Defaults to `Date()`
 	func countEnabledDays(since date: Date = Date()) -> Int {
-		Int(getContinuousEnabledInterval(since: date) / (60 * 60 * 24))
+		// 5.5 days => 6 days
+		// 0.5 days => 1 day
+		// 13.99999 days => 14 days
```

Comment:
These comments would look great as test cases :)

Reply:
Naseweis. There is already a test which is failing. :D